### ISSUE TITLE
[FR] (Another Quad Carriage discussion)

### ISSUE THREAD

**Issue by GITRDUN75** [MarlinFirmware/Marlin#21288]
First of all a big thanks to the developers. Im sure most of us have no idea the amount of time that has been invested into this project over the years.

A little background. My company is 90% ready to release our first open source 3D printer model. An IDEX welded and machined steel frame basic cartesian. Just waiting on a few things to finish up and ready to go. Prototype has been running for months and the first production model is now being tested and tuned. 
 
 Just beginning to look towards the next version which will be either a quad IDEX or toolchanger model. I much prefer the quad IDEX idea. Both have their advantages but in my opinion I find the IDEX to be superior. However it is more complex from the electronics standpoint as it requires more motors and drivers which is beyond the limits of most mother boards. But the limiting factor seems to be firmware. I have used Marlin since the Prusa Mendel days and would very much prefer to keep using it as I am very familiar with it.

 I have read through other posts about this but have not found a way to make this happen without heavy modification to the Marlin code. I know enough to tinker around in the code a bit but that much modification is probably beyond my experience. 

 So my question is, can anyone tell me if this is something that can be implemented without a major headache or is this a lost cause? I would not think it would not be a major hurdle to add quad IDEX since dual IDEX is already implemented but I could be wrong. 

 Any help is much appreciated.

**Comment by thisiskeithb** [comment]
Duplicate of https://github.com/MarlinFirmware/Marlin/issues/17441
